### DISCUSSION TITLE
is there a way to render macro in page title/navigation?

### DISCUSSION THREAD

**Discussion by jmukhammedova-ias** [fralau/mkdocs-macros-plugin#172] [Q&A] [answered]
Example: 
let's say we have a page with 
```
---
# YAML header
title: my {{ baz }}
render_macros: true
---

# {{ baz }} documentation
```
and main.py has
```
...
    env.variables['baz'] = "John Doe"
...
```

1) baz will render in `#` section (so inside the page it will show `John Doe documentation` as per sample 
2) baz will NOT render in title (therefore navigation and browser page title will both show `my {{baz}}` )
3) if I skip title completely, title will be taken from `#` section (`{{ baz }} documentation`) -  still unrendered.

Is there a way to make mkdocs render title before providing it to the browser and nav section?

Note: similar question exists in [StackOverflow](https://stackoverflow.com/questions/73273507/mkdocs-how-to-make-nav-recognize-the-title-of-a-template)

**Comment by fralau** [comment, marked as answer]
Thanks. Have you tried the latest version (1.0.2), since this aspect has been rewritten?
